src/lib/components/Alert/Alert.tsx

```diff
@@ -26,7 +26,25 @@ export interface FlowbiteAlertCloseButtonTheme {
   color: AlertColors;
 }
 
-export interface AlertColors extends Pick<FlowbiteColors, 'failure' | 'gray' | 'info' | 'success' | 'warning'> {
+export interface AlertColors
+  extends Pick<
```

Comment:
You don't need to use `Pick<>` here if you are getting all the colors from the `FlowbiteColors` interface, you just need to `AlertColors extends FlowbiteColors`. You could even, or probably remove the `AlertColors` interface, and use the `FlowbiteColors` in its place.

Reply:
Surely!

Reply:
The problem here is that I didn't added "green", "yellow" and "red" because it would be redundant to "success", "error" and "warning".

I'll add them and implement that or just replace it with an Exclude.

Reply:
Did the "redundant" way so I could implement FlowbiteColors directly. Also added the Storybook argTypes to make it easier to test.